### PR TITLE
New Dashi layout ideas

### PR DESCRIPTION
**Pull request by slazicoicr**
* Using Material UI to improve look
* All filters are now in a hidden drawer that's shown with button click
* Graphs do not update until drawer is closed
* To increase rendering speed, option exists to select which graphs to display
* Graph shows loading animation while being rendered

### CODE REVIEW

File: dashi/rnaseqc/over_time.py [hunk 1]
```diff
@@ -2,11 +2,13 @@
 import dash_core_components as dcc
 import dash_html_components as html
 import dash.dependencies as dep
+import dash.exceptions
+import itertools
 
 import plotly
+import sd_material_ui
```
Comment (callunity) [inline dashi/rnaseqc/over_time.py]: I got a `ModuleNotFoundError: No module named 'sd_material_ui'` error, does this need to be added to `setup.py`?

Comment (ijafri26) [reply dashi/rnaseqc/over_time.py]: yes, it can be installed with `pip install sd_material_ui`

Comment (slazicoicr) [reply dashi/rnaseqc/over_time.py]: It does need to be added to `setup.py`. I will do so.

Comment (slazicoicr) [reply dashi/rnaseqc/over_time.py]: @callunity I have added it to the `setup.py` file.

**Pull request merged**